resources/assets/uad_lists.json

```
@@ -31606,36 +31606,36 @@
     "removal": "Recommended"
   },
   "com.android.overlay.gmssettings": {
-    "list": "Google",
-    "description": "GMS settings overlay. REMOVING THIS MIGHT BOOTLOOP YOUR DEVICE!",
+    "list": "Oem",
+    "description": "Unused overlay to 'com.google.android.apps.safetyhub'.",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Unsafe"
+    "removal": "Recommended"
   },
   "com.android.overlay.gmstelecomm": {
-    "list": "Google",
-    "description": "GMS Telecomm. REMOVING THIS MIGHT BOOTLOOP YOUR DEVICE!",
+    "list": "Oem",
+    "description": "Overlay app to 'com.google.android.dialer', 'com.android.incallui'.",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Unsafe"
+    "removal": "Recommended"
   },
   "com.android.overlay.gmstelephony": {
-    "list": "Google",
-    "description": "REMOVING THIS MIGHT BOOTLOOP YOUR DEVICE!\nRefers to a specific package related to telephony functionality in Google Mobile Services.\nhttps://developer.android.com/reference/android/telephony/package-summary\nhttps://developer.android.com/reference/android/telecom/package-summary",
+    "list": "Oem",
+    "description": "Overlay app to 'com.google.android.dialer', 'com.google.android.gms'.",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Unsafe"
+    "removal": "Recommended"
   },
   "com.android.overlay.systemui": {
     "list": "Aosp",
-    "description": "Contain the overlay files and resources specific to the System UI customization.\nThese overlay files can be used to modify various aspects of the System UI, including the navigation bar, status bar, lockscreen, and volume dialog.\nhttps://source.android.com/docs/automotive/hmi/system_ui\nhttps://source.android.com/docs/automotive/hmi/system_ui/management_system",
+    "description": "Contain the overlay files and resources specific to the System UI customization.\nThese overlay files can be used to modify various aspects of the System UI, including the navigation bar, status bar, lockscreen, and volume dialog.\nhttps://source.android.com/docs/automotive/hmi/system_ui\nhttps://source.android.com/docs/automotive/hmi/system_ui/management_system\nOn some phones, it is an overlay to app "com.google.android.apps.safetyhub".\nCheck out this app code and think about it.",
```

```suggestion
    "description": "Contain the overlay files and resources specific to the System UI customization.\nThese overlay files can be used to modify various aspects of the System UI, including the navigation bar, status bar, lockscreen, and volume dialog.\nhttps://source.android.com/docs/automotive/hmi/system_ui\nhttps://source.android.com/docs/automotive/hmi/system_ui/management_system\nOn some phones, it is an overlay to app \"com.google.android.apps.safetyhub\".\nCheck out this app code and think about it.",
```
You need to escape it like this.